src/components/context/ProjectsProvider.js

```diff
@@ -1,74 +1,34 @@
 
-const projects = [
-  {
-    projName: "WSDR",
-    id: "p1",
-    customer: "IBM",
-    year: 2008,
-    projDesc:
-      "The objective of the WSDR application is to support the workforce management reporting using a trusted view on workforce supply.",
-    styled: false,
-  },
-  {
-    projName: "Metro",
-    id: "p2",
-    customer: "IBM",
-    year: 2009,
-    projDesc:
-      "Metro is an application that enables the perform operational optimization process within the WMI / HR process architecture.",
-    styled: false,
-  },
-  {
-    projName: "GTS Hub",
-    id: "p3",
-    customer: "IBM",
-    year: 2010,
-    projDesc:
-      "Main goal of the project: develop reports for management that graphically represent current projects state, show state of key parameters that characterize that project, and calculate/show/predict potential impact of that parameters on project cycle/life.",
-    styled: false,
-  },
-  {
-    projName: "GBS NA",
-    id: "p4",
-    customer: "IBM",
-    year: 2011,
-    projDesc:
-      "The main goal of the project is to migrate next group of reports from Brio to Cognos: Opportunities, Management, Resources, Scorecard.",
-    styled: false,
-  },
-  {
-    projName: "IPPF",
-    id: "p5",
-    customer: "IBM",
-    year: 2012,
-    projDesc:
-      "The IPPF solution provides a key resource for project managers and partners to help in managing and controlling project financial and non-financial information through Project Financial Control (planning, reconciliation and forecasting), Non-financial project assessments and Integration with back end systems.",
-    styled: false,
-  },
-  {
-    projName: "CGSP",
-    id: "p6",
-    customer: "IBM",
-    year: 2013,
-    projDesc:
-      "The latest project in the list is IBM CGSP (Client Grows Strategic Planning) provides an integrated collaborative process designed to focus on delivering revenue and signings growth and improving the overall Client Experience.",
-    styled: false,
-  },
-  {
-    projName: "Client Care",
-    id: "p7",
-    customer: "IBM",
-    year: 2015,
-    projDesc:
-      "The SmartCare tool delivers a comprehensive view of a client across all IBM divisions and geographies, providing pro-active access to trusted information and insights.",
-    styled: false,
-  },
-];
-
 const ProjectsProvider = (props) => {
-  const [myProjList, setMyProjList] = useState(projects);
+  const fetchDataHandler = async () => {
+    const response = await axios.get(
+      "https://raw.githubusercontent.com/BrunnerLivio/PokemonDataGraber/master/output.json"
+    );
+    const slicedData = response.data.slice(0, 15);
+
+    let loadedData = [];
+
+    for (const key in slicedData) {
+      loadedData.push({
+        id: slicedData[key].Number,
+        projName: slicedData[key].Name,
+        projDesc: slicedData[key].About,
+        year: slicedData[key].MaxHP,
+        customer: slicedData[key].Generation,
+        styled: false,
+      });
+    }
```

Comment:
This part might be done in functional style.
You can try to investigate on that if you want.